src/components/protein/ngl-util.js

```diff
@@ -38,3 +38,39 @@ export function init() {
     ol[ 0 ].autoView();
   });
 }
+
+export function exampleInit() {
+  // Setup to load data from rawgit
+  NGL.DatasourceRegistry.add(
+    'data', new NGL.StaticDatasource( '//cdn.rawgit.com/arose/ngl/v2.0.0-dev.32/data/' )
```

Comment:
This seems like a dev branch can we get it on a version locked one so that it doesn't get erased or changed unexpectedly.
`'//cdn.rawgit.com/arose/ngl/v2.0.0-dev.32/data/'`